libraries/ashrae/guideline36/4.10-chilled-water-plant.ttl

```diff
@@ -0,0 +1,1221 @@
+    ] ;
+    # required
+    sh:property [
+        rdfs:label "WSE CW return temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Leaving_Condenser_Water_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # required, TODO check topology, does this point belong on a demand side loop?
+    sh:property [
+        rdfs:label "CHW return temperature before WSE" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Entering_Chilled_Water_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # applications, NOTE is this implicitly required?
+    sh:property [
+        rdfs:label "CHW return temperature after WSE" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Leaving_Chilled_Water_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [ sh:node :wse-bypass-valve ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [
+            sh:or (
+                sh:node components:constant-speed-pump
+                sh:node components:variable-speed-pump
+            )
+        ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # required, TODO implicitly?
+    sh:property [
+        rdfs:label "WSE entering CHW temperature" ;
+        rdfs:comment "Note: WSE entering CHW temperatures is located in the pipe entering the HX, not the CHWR main upstream of the WSE." ;
+        sh:qualifiedValueShape [ sh:class brick:Entering_Chilled_Water_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] .
+
+:wse-isolation-valve a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Waterside Economizer Isolation Valve" ;
+    sh:class brick:Condenser_Water_Isolation_Valve ;
+    # required
+    sh:property [
+        rdfs:label "WSE HX CW isolation valve" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Off_Command ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # Retain the following two points for optional valve end switch status feedback.
+    # optional
+    sh:property [
+    rdfs:label "WSE HX CW isolation valve closed end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # optional
+    sh:property [
+        rdfs:label "WSE HX CW isolation valve open end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Off_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1
+    ] .
```

Comment:
Usually we use open/Close enumeration for the Valve and damper, and on/off is for the Fan and pumps. And the Brick https://brickschema.org/ontology/1.3/classes/Open_Close_Status will show the current valve status (open/closed)

Reply:
Ok. The G36 points are `status` points, with a note that reads...
>Retain the following two points for optional valve end switch status feedback.

Keeping as is since these are two distinct points.